<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Administrateur</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>691</width>
      <height>491</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>701</width>
      <height>491</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="CRUD">
     <attribute name="title">
      <string>GESTION ADMINISTRATEUR</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>681</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Ajout">
       <attribute name="title">
        <string>Ajout</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_AnullerAjout">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>320</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Ajouter">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>320</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NOM">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PRENOM">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_CIN">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_EMAIL">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_AGE">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SEXE">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ADRESSE">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NIVEAUETUDE">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_TEL">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>40</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>40</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>EMAIL</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>AGE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>120</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SEXE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ADRESSE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>200</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NIVEAU D'ETUDE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>200</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TEL</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Affichage">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <widget class="QTableView" name="tab_Abonnes">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>621</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Suppression">
       <attribute name="title">
        <string>Suppression</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_CIN_Suprrimer">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>CIN </string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Supprimer">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_AnnulerSupp">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_CIN_update">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>CIN </string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Modifier">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>340</y>
          <width>91</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Mettre a jour</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NOM_update">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PRENOM_update">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_TEL_update">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>290</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NIVEAUETUDE_update">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ADRESSE_update">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SEXE_update">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_AGE_updtae">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>180</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_EMAIL_update">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>180</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>110</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>160</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>EMAIL</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>160</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>AGE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SEXE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_39">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>210</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ADRESSE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_40">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>270</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NIVEAU D'ETUDE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_41">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>270</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TEL</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Entrer">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>91</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Entrer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_AnnulerModifier">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>340</y>
          <width>91</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_changerPoste">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Changer Poste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_IMPRIME">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>IMPRIMER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
